solidity/contracts/converter/ConverterVersion.sol

```diff
@@ -2,5 +2,5 @@
 pragma solidity 0.6.12;
 
 contract ConverterVersion {
-    uint16 public constant version = 45;
+    uint16 public constant version = 46;
```

Comment:
Keep in mind that the other PR is also using `46`, but since neither has been deployed yet - it's fine.

Reply:
OK.